bin/GameIndex.dbf

```diff
@@ -1657,6 +1657,17 @@ Serial = SCES-51135
 Name   = Primal
 Region = PAL-M5
 Compat = 5 // With GSdx, SW-mode only.
+vuRoundMode = 0 // Fix SPS
+[patches = dcc4eeea]
+	author=Prafull
+	//Fix slow and choppy gameplay 
```

Comment:
Minor nitpick but comments can be formatted better to suit the rest (ocd satisfaction and consistency).

`//Fix slow and choppy gameplay`
to
`// Fix slow and choppy gameplay.`

Check others too.

Reply:
Also, it would be nice to add why this patch is require.

Reply:
Ive got them from Prafull themselves. I'll ask him.